src/tools/id-mapping/components/results/__mocks__/UniProtkbMapping.ts

```diff
@@ -1,4 +1,189 @@
+          count: 4,
+        },
+        {
+          label: 'Catalytic activity',
+          value: 'catalytic_activity',
+          count: 4,
+        },
+        {
+          label: 'Chain',
+          value: 'chain',
+          count: 4,
+        },
+        {
+          label: 'Cofactors',
+          value: 'cofactor',
+          count: 4,
+        },
+        {
+          label: 'Domain',
+          value: 'domain',
+          count: 4,
+        },
+        {
+          label: 'Function',
+          value: 'function',
+          count: 4,
+        },
+        {
+          label: 'Nucleotide binding',
+          value: 'np_bind',
+          count: 4,
+        },
+        {
+          label: 'Subunit structure',
+          value: 'subunit',
+          count: 4,
+        },
+        {
+          label: '3D structure',
+          value: '3dstructure',
+          count: 2,
+        },
+        {
+          label: 'Alternative products(isoforms)',
+          value: 'alternative_products',
+          count: 2,
+        },
+        {
+          label: 'Cross-link',
+          value: 'crosslnk',
+          count: 2,
+        },
+        {
+          label: 'Binary interaction',
+          value: 'interaction',
+          count: 2,
+        },
+        {
+          label: 'Modified residue',
+          value: 'mod_res',
+          count: 2,
+        },
+        {
+          label: 'Subcellular location',
+          value: 'subcellular_location',
+          count: 2,
+        },
+        {
+          label: 'Compositional bias',
+          value: 'compbias',
+          count: 1,
+        },
+        {
+          label: 'Helix',
+          value: 'helix',
+          count: 1,
+        },
+        {
+          label: 'Beta strand',
+          value: 'strand',
+          count: 1,
+        },
+        {
+          label: 'Turn',
+          value: 'turn',
+          count: 1,
+        },
+        {
+          label: 'Alternative splicing',
+          value: 'var_seq',
+          count: 1,
+        },
+        {
+          label: 'Natural variant',
+          value: 'variant',
+          count: 1,
+        },
+      ],
+    },
+    {
+      label: 'Protein Existence',
+      name: 'existence',
+      allowMultipleSelection: true,
+      values: [
+        {
+          label: 'Protein level',
+          value: 'PROTEIN_LEVEL',
+          count: 4,
+        },
+      ],
+    },
+    {
+      label: 'Annotation Score',
+      name: 'annotation_score',
+      allowMultipleSelection: true,
+      values: [
+        {
+          value: '5',
+          count: 4,
+        },
+      ],
+    },
+    {
+      label: 'Sequence length',
+      name: 'length',
+      allowMultipleSelection: true,
+      values: [
+        {
+          label: '401 - 600',
+          value: '[401 TO 600]',
+          count: 3,
+        },
+        {
+          label: '>= 801',
+          value: '[801 TO *]',
+          count: 1,
+        },
+      ],
+    },
+  ],
```

Comment:
Could you add a comment at the top with the URL of where these values are coming from?

Also, it would be good to specify to type of the mocks in the mock files. It lets us know if there's something wrong directly here:
```
const mock: TypeOfMyMock = {...}

export default mock;
```